Index: app/src/main/java/com/example/easybus/Urls.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.easybus;\r\nimport java.net.InetAddress;\r\nimport java.net.UnknownHostException;\r\n\r\npublic class Urls {\r\n    //private local InetAddress ip IPv4\r\n    public static String url1 = \"http://192.168.27.110\";\r\n    }
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/easybus/Urls.java	(revision 27b945ac0958a0f1b4bf688ad6dfbf4c05bd7575)
+++ app/src/main/java/com/example/easybus/Urls.java	(date 1637301694878)
@@ -4,5 +4,5 @@
 
 public class Urls {
     //private local InetAddress ip IPv4
-    public static String url1 = "http://192.168.27.110";
+    public static String url1 = "http://192.168.27.111";
     }
\ No newline at end of file
